pkg/graveler/graveler.go

```diff
@@ -3050,6 +3084,27 @@ func (g *Graveler) DiffUncommitted(ctx context.Context, repository *RepositoryRe
 			return nil, err
 		}
 	}
+	if branch.CompactedBaseMetaRangeID != "" {
+		if metaRangeID == "" {
```

Comment:
How can this happen?

Reply:
I would also change it's name: `metaRangeID` -> `commitMetaRangeID`

Reply:
What do we plan to do when compaction is triggered and we have no committed data in our branch in case it's even possible?